### PR TITLE
Allow oauth proxy imagestream to be used by specifying the namespace/…

### PR DESCRIPTION
**Pull request by objectiser**
…name

### CODE REVIEW

File: pkg/cmd/start/bootstrap.go [hunk 1]
```diff
@@ -88,10 +91,61 @@ func bootstrap(ctx context.Context) manager.Manager {
...
+
+	if len(imageStream.Status.Tags) == 0 {
+		log.WithFields(log.Fields{
+			"namespace": imageStreamNamespace,
+			"name":      imageStreamName,
+		}).Error("OAuthProxy ImageStream has no tags")
+		return
+	}
+
+	image := fmt.Sprintf("%s:%s", imageStream.Status.DockerImageRepository, imageStream.Status.Tags[0].Tag)
```
Comment (rubenvp8510) [inline pkg/cmd/start/bootstrap.go:141]: Is it OK to assume that we will always use the first tag? I would say that the answer is yes, at least for now, but not sure.

**Comment by rubenvp8510** [comment]
Small comment, this looks good. 

I tested in my local environment, with and without specify an image stream and it works correctly, I was able to create all-in-one and production strategies, no seeing errors on logs.

File: pkg/cmd/start/bootstrap.go (hunk 1 above)
Comment (objectiser) [reply pkg/cmd/start/bootstrap.go:141]: Yes, the [comment on the type](https://github.com/openshift/api/blob/master/image/v1/types.go#L295-L296) indicates that the first tag is the current one.

> Tags are a historical record of images associated with each tag. The first entry in the TagEvent array is the currently tagged image

**Comment by objectiser** [comment]
@rubenvp8510 Thanks for testing this!

**Pull request merged**